src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_resource_header.scss

```diff
@@ -75,6 +75,14 @@ actions (edit, share, report, print, ...).
     a {
         color: inherit;
     }
+
+    .user-meta.user-profile-image {
+        vertical-align: -0.5em;
+        margin-left: 0.5em;
+        margin-right: 0.5em;
+        height: 1.5rem;
+        width: 1.5rem;
+    }
```

Comment:
I know. I inserted it specifically to overrule those styles. Do you know any better solution?